src/sentry_plugins/jira/plugin.py

```diff
@@ -281,12 +281,12 @@ def get_issue_label(self, group, issue_id, **kwargs):
 
     def get_issue_url(self, group, issue_id, **kwargs):
         instance = self.get_option("instance_url", group.project)
-        return "%s/browse/%s" % (instance, issue_id)
+        return u"%s/browse/%s" % (instance, issue_id)
 
     def _get_formatted_user(self, user):
-        display = "%s %s(%s)" % (
+        display = u"%s %s(%s)" % (
```

Comment:
@getsentry/python-apocalypse should I convert this `%` to `.format`?

Reply:
it's fine